backend/btrixcloud/storages.py

```diff
@@ -280,12 +271,17 @@ async def get_s3_client(
 
         session = aiobotocore.session.get_session()
 
+        config = None
+        if for_presign and storage.access_endpoint_url != storage.endpoint_url:
+            config = AioConfig(s3={"addressing_style": "virtual"})
+
         async with session.create_client(
             "s3",
-            region_name=storage.region,
+            region_name=storage.region or "any",
```

Comment:
```suggestion
            region_name=storage.region or "us-east-1",
```
Switch back to us-east-1